Jenkinsfile

```diff
@@ -35,13 +35,39 @@ node('linux && docker') {
         junit 'packages/*/reports/*.xml'
       }
 
-      stage('Cypress Test') {
+      stage('Prepare Cypress') {
         sh 'apt-get -y install libgtk2.0-0 libgtk-3-0 libnotify-dev libgconf-2-4 libgbm-dev libnss3 libasound2 xauth xvfb'
         sh 'rm -rf /var/lib/apt/lists/*'
-        sh 'cd packages/atomic && ./node_modules/cypress/bin/cypress install'
-        sh 'cd packages/atomic && npm run start:prod & npx wait-on http://localhost:3333'
-        sh 'chown -R $(whoami) /tmp'
-        sh 'cd packages/atomic && NO_COLOR=1 ./node_modules/cypress/bin/cypress run --record --key 0e9d8bcc-a33a-4562-8604-c04e7bed0c7e --browser chrome'
+      }
+
+      stage('Cypress Test') {
+        parallel([
+          'atomic': {
+            sh 'cd packages/atomic && ./node_modules/cypress/bin/cypress install'
```

Comment:
We are installing cypress twice, perhaps we can move it at the root and install it once, or is it better to keep 2 separate versions of it?

Reply:
I personally like that atomic and quantic can use their own Cypress version. I wonder if moving Cypress at the root would work since we need to override the `HOME` environment variable for quantic...